tests/GeneratorTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
```

Comment:
We should have strict typing turned on in this file.

Reply:
Resolved.